react/components/InputPassword/index.js

```diff
@@ -18,7 +18,14 @@ class InputPassword extends Component {
     showPassword: false,
   }
 
-  toggle = () => this.setState(state => ({ showPassword: !state.showPassword }))
+  handleToggle = () =>
+    this.setState(state => ({ showPassword: !state.showPassword }))
+
+  handleKeyDown = event => {
+    if (event.key === 'Enter') {
+      this.handleToggle()
+    }
+  }
```

Comment:
The button has the behavior of triggering the event when pressed Space or Enter, in this case only Enter is resolved.

This solves the problem and makes reading more readable.

```suggestion
  handleKeyDown = event => {
    const SPACE = ' '
    const ENTER = 'Enter'

    if (key === SPACE || key === ENTER) {
        this.handleToggle()
    }
  }
```